shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -667,13 +667,15 @@ wrapperName
     ;
     
 functionName
-    : identifier
-    | (owner DOT_)? identifier
+    : (owner DOT_)? identifier
     ;
-    
+
+procName
```

Comment:
What statement corresponds to the procName parsing rule?

Reply:
`SHOW CREATE PROCEDURE procName`

Reply:
Do you think `procedureName` is better?

Reply:
Yes!